pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -417,6 +417,10 @@ type PipelineRunStatusFields struct {
 	// +optional
 	// +listType=atomic
 	ChildReferences []ChildStatusReference `json:"childReferences,omitempty"`
+
+	// FinallyStartTime is when all non-finally tasks have been completed and only finally tasks are being executed.
```

Comment:
I'm wondering if there's a way to look at any of the finally TaskRuns and get their start time? Or would this introduce more race conditions?

Reply:
I thought about this, but at best it'd be messy, and I can't rule out the possibility of race conditions, inconsistent behavior, etc, so I opted for this. It's not bad information to have in general, I think - the dashboard or CLI or some other downstream consumer of `PipelineRun`s can just look at this field (plus `StartTime` and `CompletionTime`) to see how long just tasks took and how long finally tasks took, etc.